oop/js/Fibonacci.js

```diff
@@ -21,8 +50,15 @@ document.getElementById("form").addEventListener("submit", function (event) {
     const sequence = event.target["sequence"].value;
     const method = event.target["method"].value;
 
-    const result = new Fibonacci(sequence);
-
+    let result = [];
+    const resultObj = new Fibonacci(sequence);
+    if (method === "loop") {
+      result = resultObj.generateFibonacciUsingLoop(sequence);
+    } else if (method === "recursive") {
+      result = resultObj.generateFibonacciUsingRecursive(sequence);
+    } else {
+      throw new Error("Method must be loop or recursive.");
+    }
```

Comment:
We should keep our main code clear. So I think this "method" checking can be organized into one method on the Fibonacci class.